### ISSUE TITLE
Change container size?

### ISSUE THREAD

**Issue by Gamerzfanz21** [darktrojan/newtabtools#453]
Is there a way to resize the thumbnail containers to a different size?  (The "boxes" that the thumbnails reside in)
![thumbnail_containers](https://user-images.githubusercontent.com/22859500/58751902-66a97d80-8473-11e9-9d53-0bf2f6a1718d.png)

**Comment by DanaMW** [comment]
Changing the number of rows and margins in the settings is the intended way . The code that draws (styles) them is newtab.css in his extension.

**Comment by Gamerzfanz21** [comment]
Yeah, I figured out how to edit that file, but I'm unable to "resave" and overwrite, as it gives an error about not having a valid manifest.  So hopefully in the future, there will be another way to change those sizes through options.
![blur](https://user-images.githubusercontent.com/22859500/58853735-24e22800-8669-11e9-9f77-0c7801ff09f3.png)

**Comment by DanaMW** [comment]
If you use developers version of firefox you can include the changes in your userContent.css to over ride it. If you use chrome developers you can unzip it, remove the manifest (key) and load it as an unpacked extension. Lots of docs online about doing both,
